packages/dashboard/src/components/infiniteScroller/stories/index.js

```diff
@@ -82,14 +91,7 @@ export const _default = () => {
           }}
```

Comment:
💭 for the future would be good to add a timeout here so people can see the loading message